js/controllers/SiteHandler.js

```diff
@@ -39,6 +39,10 @@
 
 		$('.go').button();
 
+		this.bindGlobal('reloadPage', function () {
```

Comment:
Is this used anywhere? I don't see it.

Reply:
You are right @asmecher. This code was used at a previous version of the feature, in order to reload the submission page upon the change of the permitMetadataEdit option for a participant. 

Removed.